compact.go

```diff
@@ -514,8 +562,11 @@ func (c *LeveledCompactor) write(dest string, meta *BlockMeta, blocks ...BlockRe
 	}
 	defer indexw.Close()
 
-	if err := c.populateBlock(blocks, meta, indexw, chunkw); err != nil {
+	if overlapping, err := c.populateBlock(blocks, meta, indexw, chunkw); err != nil {
```

Comment:
why not use the logger and metrics directly in `c.populateBlock` ?

Reply:
Done